RELEASE.md

```diff
@@ -6,10 +6,7 @@ Please follow the established format:
 - Include the ID number for the related PR (or PRs) in parentheses
 -->
 
-# Release 10.2.0 
-
-
-## Major features and improvements
+# Release 10.2.0
```

Comment:
Should we add a more prominent note on the changelog about the deprecation of Experiment Tracking? Maybe reusing part of the text of the modal

Reply:
100%; btw Juanlu we have a deprecation warning on Kedro in the CLI wherein we say tracking datasets will be deprecated but we don't have any CLI warnings on Kedro, or Kedro-viz. Do we want to add that on Kedro-viz CLI, if so we will do a quick PR.

Reply:
Let me know if the changes look fine.